crates/revm/src/inspector/eip3155.rs

```diff
@@ -98,12 +100,14 @@ impl TracerEip3155 {
 }
 
 impl TracerEip3155 {
-    pub fn new(output: Box<dyn Write>, print_summary: bool) -> Self {
+    pub fn new(output: Box<dyn Write>, print_summary: bool, include_memory: bool) -> Self {
```

Comment:
Wound not add include_memory here flag here  but a new function `with_memory(self) -> Self`

This would allow setting `TracerEip3155` as TracerEip3155::new(Box::new(std::io::stdout()), true).with_memory()`

Would be good to add same for `print_summary` but your choice if you want it in this PR

Reply:
Thanks for the suggestion, I've implemented it for both now.
For the summary I've defaulted to true, as it is a MUST for the (current) EIP-3155. This implicitly adds it to the revme `execute_test_suite` stderr and stdout. I could change that if that's not wanted.

For the memory I've defaulted to false, as it's not mandatory and often not wanted due to performance issues.